src/test/resources/addSuggestDocsWithContext.csv

```diff
@@ -0,0 +1,3 @@
+text,weight,payload,context
+the cat meows,1,payload1,lucene
```

Comment:
Not sure if you want to support multiple contexts (and context types) in this review.

I think here context should be a list or a map <string, List> :
could could add 2 geocontext:
geocontext1: {precision:5, boost: 3}
geocontext2: {precision:4, boost: 2}

Another context, using yelp use case as an example, would be the categorie of the a biz:

catcontext1: { names [restaurant, cafes], boost: 3}



I coul

Reply:
Lucene seems to  currently support contexts only on AnalyzingInfixSuggester.  Also it seems like it supports only store field for context. It makes term queries for all values on this field with a SHOULD clause.  https://github.com/apache/lucene-solr/blob/master/lucene/suggest/src/java/org/apache/lucene/search/suggest/analyzing/AnalyzingInfixSuggester.java#L537

If we want to support multiple contexts we might need to implement our own Lookup and also our own InputIterator (which itself contains the contexts to be built out in the first place). This is doable and we could probably ticket this if needed.

Reply:
Yes I think we should support it. Can make a ticket and resolve

Reply:
added issue. resolving this for now.